docs/tutorials/java-troubleshooting.rst

```diff
@@ -0,0 +1,33 @@
+Java Troubleshooting
+######################################################
+
+In this tutorial, you will use Robusta to troubleshoot a java application running on Kubernetes. Specifically,
+you will be remotely running functions from a custom JDK on any Java application.
```

Comment:
Don't like the second sentence here. It describes the low-level details for *how it works* but the user cares more about the *why*. Why should you want to follow this tutorial? What problems will it solve?